oqs/__init__.py

```diff
@@ -1,5 +1,5 @@
-from oqs.wrapper import _enabled_KEMs, _supported_KEMs, KeyEncapsulation
-from oqs.wrapper import _enabled_sigs, _supported_sigs, Signature
+from oqs.wrapper import _enabled_KEMs, _supported_KEMs, OQS_KEM
```

Comment:
Why are `_enabled_KEMs`, `_supported_KEMs`, `_enabled_sigs`, `_supported_sigs` available as top level objects?  Can we just have them as static methods on OQS\_KEM / OQS\_SIG?

Reply:
I've tinkered a bit with this. There's some fiddly stuff with scopes and the liboqs ctypes var, and I've deferred it for later.